web/src/settings_invites.js

```diff
@@ -289,7 +289,10 @@ export function update_invite_users_setting_tip() {
 
 export function update_invite_user_panel() {
     update_invite_users_setting_tip();
-    if (!settings_data.user_can_invite_others_to_realm()) {
+    if (
+        !settings_data.user_can_invite_others_to_realm() &&
```

Comment:
Would also be good to rename `user_can_invite_others_to_realm` to something like `user_can_send_email_invite_to_others`.

Reply:
Should I just rename this function or all the variables related to this setting?

Reply:
I think both because having `can_invite_others_to_realm` as `False`, feels user is not allowed to create multiuse link as well. Would probably be better to do this change in a separate prep commit.

Reply:
Ohh, sorry for the confusion here. But I did not meant to rename the database field and the API fields sent through `/events`, `/register`,etc. endpoints. I only meant the variables used internally. It would be better to get feedback from others before renaming the database fields and the fields sent in API responses.

Reply:
I have updated the PR. Please have a look.